gallery/src/styles/NxList/NxListExamples.tsx

```diff
@@ -59,6 +60,14 @@ const NxListsExamples = () =>
       default <NxCode>&lt;ul&gt;</NxCode> styling. These can be nested.
     </GalleryExampleTile>
 
+    <GalleryExampleTile title="NX Numbered List Example"
+                        id="nx-list-numbered-example"
+                        htmlExample={NxListNumberedCode}
+                        codeExamples={NxListNumberedCode}>
+      An <NxCode>nx-list</NxCode> demonstrating ordered (numbered) list styles more typical of
+      default <NxCode>&lt;ol&gt;</NxCode> styling. These can be nested.
```

Comment:
Is there a limit to how far they can be nested?

Reply:
> Is there a limit to how far they can be nested?

Like bulleted lists we aren't putting a limit on nesting, but we have only explicitly styled the first 3 levels

Reply:
> Like bulleted lists we aren't putting a limit on nesting, but we have only explicitly styled the first 3 levels

Should we at least have some awareness of what it does if you go deeper?

Reply:
> > Like bulleted lists we aren't putting a limit on nesting, but we have only explicitly styled the first 3 levels
> 
> Should we at least have some awareness of what it does if you go deeper?

Like the bulleted list it just uses the last defined `list-type` for all nested list items past the third. So in this case that's Roman numerals (I also verified this in the code).